examples/Dialog.draggable.jsx

```diff
@@ -12,38 +11,26 @@ import {
 } from '@itwin/itwinui-react';
 
 export default () => {
-  const [isOpen, setIsOpen] = React.useState(false);
+  const dialog = Dialog.useInstance();
 
   return (
     <>
-      <Button styleType='high-visibility' onClick={() => setIsOpen(true)}>
+      <Button styleType='high-visibility' onClick={dialog.show}>
```

Comment:
Will the event argument being passed to the show function not cause issues?

```suggestion
      <Button styleType='high-visibility' onClick={() => dialog.show()}>
```

If not, then can this be done for `dialog.close` too?

Reply:
While it shouldn't cause any issues, I've still changed it for consistency.

Reply:
done in [`76c9556`](https://github.com/iTwin/iTwinUI/pull/1983/commits/76c9556b7fb515383a044c00f8d736f2bd5ae99c)